easybuild/easyconfigs/f/freebayes/freebayes-1.3.5_find-external-libs.patch

```diff
@@ -0,0 +1,49 @@
+diff -Nru freebayes-1.3.5.orig/meson.build freebayes-1.3.5/meson.build
```

Comment:
Could you please add a short description + a note of who the author is (probably you?) in all the patches?

Reply:
done